localization/english/event_localization/activities/wc_activities_l_english.yml

```diff
@@ -0,0 +1,3 @@
+﻿l_english:
+ ##### PILGRIMAGE #####
+ activity_pilgrimage.war_dangerous:0 "It's too dangerous to travel here right now"
```

Comment:
Bad idea to put everything in a file, it's better to separate what needs to be separated into the corresponding files.
We have lots of messy files because of that.